sources/dynamodb/toddl.go

```diff
@@ -15,67 +15,21 @@
 package dynamodb
 
 import (
-	"fmt"
-	"strconv"
-	"unicode"
-
 	"github.com/cloudspannerecosystem/harbourbridge/internal"
 	"github.com/cloudspannerecosystem/harbourbridge/schema"
 	"github.com/cloudspannerecosystem/harbourbridge/spanner/ddl"
 )
 
-// schemaToDDL performs schema conversion from the source DB schema to
-// Spanner. It uses the source schema in conv.SrcSchema, and writes
-// the Spanner schema to conv.SpSchema.
-func schemaToDDL(conv *internal.Conv) error {
-	for _, srcTable := range conv.SrcSchema {
-		spTableName, err := internal.GetSpannerTable(conv, srcTable.Name)
-		if err != nil {
-			conv.Unexpected(fmt.Sprintf("Couldn't map source table %s to Spanner: %s", srcTable.Name, err))
-			continue
-		}
-		var spColNames []string
-		spColDef := make(map[string]ddl.ColumnDef)
-		conv.Issues[srcTable.Name] = make(map[string][]internal.SchemaIssue)
-		// Iterate over columns using ColNames order.
-		for _, srcColName := range srcTable.ColNames {
-			srcCol := srcTable.ColDefs[srcColName]
-			colName, err := internal.GetSpannerCol(conv, srcTable.Name, srcCol.Name, false)
-			if err != nil {
-				conv.Unexpected(fmt.Sprintf("Couldn't map source column %s of table %s to Spanner: %s", srcTable.Name, srcCol.Name, err))
-				continue
-			}
-			spColNames = append(spColNames, colName)
-			ty, issues := toSpannerType(conv, srcCol.Type.Name, srcCol.Type.Mods)
-
-			if len(issues) > 0 {
-				conv.Issues[srcTable.Name][srcCol.Name] = issues
-			}
-			spColDef[colName] = ddl.ColumnDef{
-				Name:    colName,
-				T:       ty,
-				NotNull: srcCol.NotNull,
-				Comment: "From: " + quoteIfNeeded(srcCol.Name) + " " + srcCol.Type.Print(),
-			}
-		}
-		comment := "Spanner schema for source table " + quoteIfNeeded(srcTable.Name)
-		conv.SpSchema[spTableName] = ddl.CreateTable{
-			Name:     spTableName,
-			ColNames: spColNames,
-			ColDefs:  spColDef,
-			Pks:      cvtPrimaryKeys(conv, srcTable.Name, srcTable.PrimaryKeys),
-			Indexes:  cvtIndexes(conv, spTableName, srcTable.Name, srcTable.Indexes),
-			Comment:  comment}
-	}
-	return nil
+// ToDdl implementation for DynamoDB
```

Comment:
to delete these changes once the previous PR is merged

Reply:
These will automatically go away from the diff once the previous PR is merged. No action needed here.